app/views/authorisationsAndLimit/authorisations/index/AuthorisationReferenceNumberView.scala.html

```diff
@@ -49,7 +49,8 @@
                     HtmlFormat.fill(
                         Seq(
                             if (authType.isACR) Some(messages("authorisations.authorisationReferenceNumber.paragraph.ACR").toParagraph) else None,
-                            if (authType.isTRD) Some(messages("authorisations.authorisationReferenceNumber.paragraph.TRD").toParagraph) else None
+                            if (authType.isTRD) Some(messages("authorisations.authorisationReferenceNumber.paragraph.TRD").toParagraph) else None,
+                            if (authType.isSSE) Some(messages("authorisations.authorisationReferenceNumber.paragraph.SSE").toParagraph) else None
```

Comment:
It looks like this `HtmlFormat.fill` code is a hangover from when the logic was like [this](https://github.com/hmrc/ctc-departure-transport-details-frontend/blob/c07055893aa75be4f3fad17999c890c8f419caa8/app/views/authorisationsAndLimit/authorisations/index/AuthorisationReferenceNumberView.scala.html).

Given that only one of these can be true we could look to refactor to something like
```
additionalHtml = authType match {
                    case x if x.isACR => Some(messages("authorisations.authorisationReferenceNumber.paragraph.ACR").toParagraph)
                    case x if x.isTRD => Some(messages("authorisations.authorisationReferenceNumber.paragraph.TRD").toParagraph)
                    case x if x.isSSE => Some(messages("authorisations.authorisationReferenceNumber.paragraph.SSE").toParagraph)
                    case _            => None
                }
```